packages/cursorless-engine/src/languages/LanguageDefinitions.ts

```diff
@@ -50,4 +52,11 @@ export class LanguageDefinitions {
 
     return definition === LANGUAGE_UNDEFINED ? undefined : definition;
   }
+
+  /**
+   * @deprecated Only for use in legacy containing scope stage
+   */
+  public getNodeAtLocation(document: TextDocument, range: Range): SyntaxNode {
+    return this.treeSitter.getNodeAtLocation(document, range);
+  }
```

Comment:
I wasn't sure what the best approach was, as this function won't be used for new query scopes.  Seemed fine to put it here and mark it as deprecated